### ISSUE TITLE
bad default genesis

### ISSUE THREAD

**Issue by Sen-Com** [White-Whale-Defi-Platform/migaloo-chain#149]
The default genesis.json does not work out of the box. It contains multiple "stake" denom references that make lead to crashes.

## Steps to Reproduce
$binary init testnode --chain-id migaloo-1
$binary add-genesis-account test 1000000000000uwhale
$binary gentx test 1000000000uwhale --chain-id migaloo-1
$binary collect-gentxs
$binary validate-genesis

**Comment by faddat** [comment]
nice!

We'll get this fixed asap, thanks for the report :)

**Comment by vuong177** [comment]
@Sen-Com if you want to change your denom, you also need to change all denom in the genesis file.

**Comment by faddat** [comment]
@vuong177 I don't think that is the issue.  Lemme dig for a second.  @Sen-Com is there any need to apply this fix to v2?
